### PR TITLE
Alz 104

### PR DESCRIPTION
**Pull request by j-hendrickson-sage**
Updates to readme, works for local development

### CODE REVIEW

File: app.R [hunk 1]
```diff
@@ -2,9 +2,17 @@
...
+    options(shiny.port = 8100)
+  } else {
+    Sys.setenv(R_CONFIG_ACTIVE = "default")
+  }
 }
+
 pkgload::load_all()
 options("golem.app.prod" = TRUE)
-stopadforms::run_app() # add parameters here (if any)
+stopadforms::run_app() # add parameters here (if any)
```
Comment (JessterB) [inline app.R:18]: do you think we need a new line at the end of this file?

File: README.md [hunk 2]
```diff
@@ -67,11 +70,14 @@ submissions that are stored in Synapse.
 Running the application locally
 -------------------------------
 
+Set localDevelopment variable to "TRUE" in "app.R" file of main directory.
+
 ``` r
-library("stopadforms")
-run_app()
+localDevelopment = TRUE
 ```
```
Comment (JessterB) [inline README.md:78]: Add info here about the config file you need for local development too! You can either just document it here, or also check in a 'clean' version of it (no secrets) that people can populate locally.

**Review by JessterB — CHANGES_REQUESTED** [review]

File: app.R (hunk 1 above)
Comment (j-hendrickson-sage) [reply app.R:18]: I don't think so for this an R file.   There needs to be a new line for .yml files, though.  All operating systems pass.

File: README.md (hunk 2 above)
Comment (j-hendrickson-sage) [reply README.md:94]: added clean config.yml file

**Review by JessterB — APPROVED** [review]
Looks good!

**Review by andrewelamb — APPROVED** [review]

**Pull request merged**